xdem/ddem.py

```diff
@@ -157,7 +157,7 @@ def interpolate(self, method: str = "linear",
                         interpolated_ddem,
                         reference_elevation.data,
                         mask=feature_mask
-                    ).data
+                    )#.data   # This causes an issue at l 172: NotImplementedError: sub-views are not implemented
```

Comment:
This entire statement should probably be encased in a `np.asarray(...)` instead. `xdem.volume.hypsometric_interpolation()` should return a masked_array (but we want ndarrays in this case), which is why I took the data property. 

Unfortunately, `np.ndarray.data` is something completely different from `np.ma.masked_array.data`!!